markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+    latex_engine: xelatex
+---
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Version number: 3
+
+# Template: Code repository catalogue
+
+1. Create new project: Type script
+2. Screenshot of the \emph{complete} first page
+3. For a published paper, prepare the link to the repository. For a WP prepare a zip that contains all data and code; For \textbf{DIME verified} packages also the reproducibility verification statement
+
+# Information on metadata
+This is information about the creator of the metadata. Sufficient fields
+\begin{enumerate}
+  \item Name: Second name only: Smith
+  \item Abbreviation: PS
+  \item Name: Smith
+  \item Affiliation: World Bank
+  \item Production date: Year-Month-Day
+  \item Version number: Entered as 1,2,3
+\end{enumerate}
+
+# Project description
+## Title statement
+Sufficient fields are
+\begin{enumerate}
+  \item Primary ID: Ideally a DOI or an organization consistent scheme;
+      \begin{enumerate}
+        \item Published Paper: 
+            \begin{enumerate}
+              \item A corresponding PRWP exists: FORMAT
+                \begin{lstlisting}
+                  PP_CountryCodeIso3_Year_PRWP-PRWPno_versionno
+                  Example: PP_CRI_2018_PRWP-8524_v01
+                \end{lstlisting}
+              \item A corresponding PRWP does not exit: FORMAT
+                \begin{lstlisting}
+                  PP_CountryCodeIso3_Year_EXT-EXTno_versionno
+                  Example: PP_CRI_2018_EXT-1_v01
+                \end{lstlisting}
+                    EXTno is a unique running number assigned by Dime.
+            \end{enumerate}
+        \item Working Paper: FORMAT
+          \begin{lstlisting}
+            RR_CountryCodeIso3_Year_PRWP-PRWPno_versionno
+            RR_CRI_2018_PRWP-8525_v01
+          \end{lstlisting}
+        \end{enumerate}
+  \item Other identifiers: DOI and https://doi.org/number
+  \item Title: Of the following form
+    \begin{lstlisting}
+      Reproducibility package for TITEL(paper)
+    \end{lstlisting}
+  \item Subtitle:
+    \begin{itemize}
+      \item WP: World Bank Policy Research Working Paper (PRWP) number
+      \item PP: DOI number (no https)
+    \end{itemize}
+\end{enumerate}
+
+## Authors and contributors
+Sufficient are
+\begin{enumerate}
+    \item Authoring entity: author, affiliation and ORCID number and email when available; For world bank authors use \textbf{World Bank}. Example:
+      \begin{verbatim}
+        Christoph Lakner World Bank ORCID 0000, clakner@worldbank.org
+      \end{verbatim}
+    \item Contributors: like RAs (not conference participants). Also coeditors if specifically mentioned. FORMAT
+    \begin{lstlisting}
+      Coeditor
+    \end{lstlisting}
+    \item Sponsors: All financial contributions and individual/general grants
+    \item Acknowledgements: Usually empty; Example: Corporate Taxation: special mentioning ministry employees for data assistance (could go to acknowledgment statement)
```

Comment:
suggest removing, or making a much clearer case for what goes here that would not be included in acknowledgement statement.

Reply:
OK. This is ambiguous. Perhaps best directly not to use this field.